src/database/index.ts

```diff
@@ -10,10 +10,7 @@ export function getClient(): PrismaClient {
     return client;
 }
 
-export async function start() {
-    // test our connection
-    await client.$connect();
-}
+export const connectToDatabase = async () => await client.$connect();
```

Comment:
Although this is a single line bodied function, more will be implemented here for the modmail system (#104 )

Reply:
Sounds good, although I made it a single line function, the main reason for this change was to define the intent of the function a little better.

Reply:
Ah, eventuall , there will be a game loop (setInterval) here that deletes expired messages in the modmail system.